src/components/FilterBar.jsx

```diff
@@ -21,14 +30,52 @@ const amounts = {
   5000: '> 5000',
 };
 
-function FilterBar() {
+const sortingOptions = {
+  deadlineSoon: 'Deadline (Earliest to Latest)',
+  deadlineLatest: 'Deadline (Latest to Earliest)',
+  amountLow: 'Amount (Low to High)',
+  amountHigh: 'Amount (High to Low)',
+};
+
+export default function FilterBar(props) {
+  const { passedFunction } = props;
+
+  function modifySorting(sortOption) {
+    switch (sortOption) {
+      case 'one':
+        return passedFunction('deadline');
+      case 'two':
+        return passedFunction('deadlineReverse');
+      case 'three':
+        return passedFunction('amount.min');
+      case 'four':
+        return passedFunction('amount.max');
+      default:
+        return null;
```

Comment:
What's with `one`, `two`, `three` and `four`? Why not just have `modifySorting` take in the string your going to pass along anyways so you can do:
```jsx
function modifySorting(sortOption) {
  return passedFunction(sortOption);
}
```
In fact, you get rid of `modifySorting` completely by passing in `passedFunction` instead of `modifySorting`

Reply:
Punted for #342